src/Auth0Client.ts

```diff
@@ -678,10 +686,59 @@ export default class Auth0Client {
       ...tokenResult,
       decodedToken,
       scope: params.scope,
-      audience: params.audience || 'default',
+      audience: params.audience || 'default'
     };
   }
 
+  /**
+   * ```js
+   * auth0.logout();
+   * ```
+   *
+   * Clears the application session and performs a redirect to `/v2/logout`, using
+   * the parameters provided as arguments, to clear the Auth0 session.
+   * If the `federated` option is specified it also clears the Identity Provider session.
+   * If the `localOnly` option is specified, it only clears the application session.
+   * It is invalid to set both the `federated` and `localOnly` options to `true`,
+   * and an error will be thrown if you do.
+   * [Read more about how Logout works at Auth0](https://auth0.com/docs/logout).
+   *
+   * @param options
+   */
+  public async logoutSilently(options: LogoutSilentlyOptions) {
```

Comment:
Can we move this up the file and group it underneath the default `login` method?

Reply:
Also, this function should be able to take no arguments and default to an empty object, just like `logout`. Currently I get an error when I pass no arguments.

Can we change that, and also make sure there is a test around that scenario?